src/std/mod.rs

```diff
@@ -399,6 +404,39 @@ where
     }
 }
 
+fn convert_fee_details(details: FeeDetails<NumberOrHex>) -> ApiResult<FeeDetails<u128>> {
+    let inclusion_fee = inclusion_fee_with_balance(details.inclusion_fee)?;
+    let tip = details
+        .tip
+        .try_into()
+        .map_err(|_| ApiClientError::TryFromIntError)?;
+    Ok(FeeDetails { inclusion_fee, tip })
+}
+
+fn inclusion_fee_with_balance(
+    inclusion_fee: Option<InclusionFee<NumberOrHex>>,
+) -> ApiResult<Option<InclusionFee<Balance>>> {
+    let inclusion_fee = if let Some(inclusion_fee) = inclusion_fee {
+        Some(InclusionFee {
+            base_fee: inclusion_fee
+                .base_fee
+                .try_into()
+                .map_err(|_| ApiClientError::TryFromIntError)?,
```

Comment:
Awful, due to the real `TryFromIntError` not implementing `Debug` (it's only a `()`!) , so I can't add it properly to `ApiClientError`

Reply:
Hahah, I feel you.